pom.xml

```diff
@@ -47,7 +47,7 @@
     <maven.install.version>2.5.2</maven.install.version>
     <maven.jar.version>3.0.2</maven.jar.version>
     <maven.surefire.version>2.22.1</maven.surefire.version>
-    <antlr.version>4.8-1</antlr.version>
+    <antlr.version>4.10.1</antlr.version>
```

Comment:
confirmed that they changed their versioning to the standard format.  https://search.maven.org/artifact/org.antlr/antlr4/4.10.1/jar